backend_test.go

```diff
@@ -121,10 +125,10 @@ func (e *Env) CreateConfig(t *testing.T) {
 		Path:      "config",
 		Storage:   e.Storage,
 		Data: map[string]interface{}{
-			"certificates": e.TestConf.Certificates,
-			"pcf_api_addr": e.TestConf.PCFAPIAddr,
-			"pcf_username": e.TestConf.PCFUsername,
-			"pcf_password": e.TestConf.PCFPassword,
+			"identity_ca_certificates": e.TestConf.IdentityCACertificates,
```

Comment:
I changed this field name to increase clarity around what it's looking for. Cloud Foundry has quite a lot of CA certificates.